diff --git a/dist/schema.js b/dist/schema.js
index 3d0adfcc7f3126c20a966d4a19d65b3e5f722321..a8cd0791559ac7e33b0783f0d38cde1982a45e0b 100644
--- a/dist/schema.js
+++ b/dist/schema.js
@@ -202,9 +202,9 @@ const Purpose = new graphql_1.GraphQLEnumType({
     },
 });
 const $core = new graphql_1.GraphQLDirective({
-    name: '@core',
+    name: '_core',
     args: {
-        feature: { type: graphql_1.GraphQLNonNull(graphql_1.GraphQLString), },
+        feature: { type: new graphql_1.GraphQLNonNull(graphql_1.GraphQLString), },
         as: { type: graphql_1.GraphQLString },
         'for': { type: Purpose }
     },